app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -320,12 +320,13 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
         );
       }
       final patchsets = jsonResponseTriage['patchsets'] as List<dynamic>;
+      // Note: there can be multiple patchsets with the same id, ensure that
+      // all are collected.
       var untriaged = 0;
       for (var i = 0; i < patchsets.length; i++) {
         final patchset = patchsets[i] as Map<String, dynamic>;
         if (patchset['patchset_id'] == pr.head!.sha) {
-          untriaged = patchset['new_untriaged_images'] as int;
-          break;
+          untriaged += patchset['new_untriaged_images'] as int;
```

Comment:
I wonder if this might create a situation where,

```
patchset 1 has 5 untriaged images

:make some code changes, fix regression whatever:

patchset 2 has 0 untriaged images

:flutter-gold reports images changes anyways:
```

I don't know if Gold reconciles that from patchset to patchset. I think the report back we get is just, this is what Gold saw from change to change. So I guess maybe we'll just see?

Reply:
but we're filtering down by patchset id = latest sha anyway. So this should only change the results if there are multiple patchsets with the same id and that Id is the head commit